data/etc/apache2/conf-available/repro-plinth.conf

```diff
@@ -4,7 +4,5 @@
 ##
 <Location /repro>
     ProxyPass http://localhost:5080
-
-    Include includes/freedombox-auth-ldap.conf
-    Require ldap-group cn=admin,ou=groups,dc=thisbox
+    Include includes/auth-pubtkt.conf
```

Comment:
Is it possible to keep the group restriction, so only admins can access the repro configuration panel?